libnavigation-metrics/src/main/java/com/mapbox/navigation/metrics/internal/utils/extentions/MetricEvent.kt

```diff
@@ -0,0 +1,54 @@
+package com.mapbox.navigation.metrics.internal.utils.extentions
+
+import androidx.annotation.StringDef
+import com.google.gson.Gson
+import com.mapbox.android.telemetry.Event
+import com.mapbox.navigation.metrics.internal.NavigationAppUserTurnstileEvent
+
+interface MetricEvent {
```

Comment:
Do we need this? This seems duplicated from the one we have in `libnavigation-base` 👀 https://github.com/mapbox/mapbox-navigation-android/blob/c325d9d2531a008f87cc14945edf69c3daf2f602/libnavigation-base/src/main/java/com/mapbox/navigation/base/metrics/MetricEvent.kt#L6 Can't we use that?

Reply:
There is no 'metrics' package in libnavigation-base. Based on prior PR comments all metrics related code is either in libnavigation-core or in libnavigation-metrics.